src/node/eviction.h

```diff
@@ -38,8 +38,10 @@ struct NodeEvictionCandidate {
  * fixed numbers of desirable peers per various criteria, followed by (mostly)
  * ratios of desirable or disadvantaged peers. If any eviction candidates
  * remain, the selection logic chooses a peer to evict.
+ * @param[in]   force   Attempt to evict a random peer if no candidates
+ *                      are available. (see CConman::AttemptToEvictConnection())
```

Comment:
c5e7e6563201965e88e07aa8da8f26e17fc1b4db

"among inbound no-noban connections" or something (i hate `no-noban` but not sure what's better)

Reply:
done, thanks